controllers/ebpf/internal/permissions/permissions.go

```diff
@@ -72,7 +72,7 @@ func (c *Reconciler) reconcileNamespace(ctx context.Context) error {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: c.privilegedNamespace,
 			Labels: map[string]string{
-				"app":                                "network-observability-operator",
+				"app":                                constants.OperatorName,
```

Comment:
Should it be `netobserv-operator` or `netobserv` ?

Reply:
Actually I never realized the privileged namespace had an app label ... @mariomac do you remember why is that? Is it needed?

Reply:
As far as I remember it was just to follow [k8s recommendations](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels)

We also talked about adding `app.kubernetes.io/part-of` & `app.kubernetes.io/version` in the past.

Maybe it's the right time to do it ?

Reply:
Yeah as part as I remember it was just for self-documentation metadata. Something like "this namespace belongs to this app", even if it is clear enough in the namespace name.